packages/application-shell/src/components/quick-access/butler-container/butler-container.js

```diff
@@ -1,41 +1,14 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import omit from 'lodash.omit';
 import styles from './butler-container.mod.css';
 
-class ButlerContainer extends React.Component {
-  static displayName = 'ButlerContainer';
-  static propTypes = {
-    timedOut: PropTypes.bool,
-    error: PropTypes.any,
-  };
-
-  componentDidUpdate() {
-    if (this.props.error) {
-      // eslint-disable-next-line no-console
-      console.error('Failed to load component', this.props.error);
-    }
-  }
-
-  render() {
-    // render no overlay in case of loding error, just show nothing then
-    if (this.props.timedOut || this.props.error) return null;
-
-    return (
-      <div
-        className={styles.container}
-        tabIndex="-1"
-        // omit this.props of react-loadable
-        {...omit(this.props, [
-          'error',
-          'timedOut',
-          'pastDelay',
-          'isLoading',
-          'retry',
-        ])}
-      />
-    );
-  }
-}
+const ButlerContainer = React.memo(props => (
```

Comment:
...why not 😁

Reply:
Not too sure, but wouldn't this prevent updates within `Butler` like `shouldComponentUpdate` does?

Reply:
Tho component is only used for the loading state

Reply:
Oh shit, I see now it's also used in the `<Butler>`. Sorry my mistake, will remove the memoization

Reply:
Alright, I ended up removing the intermediate `<ButlerContainer>` as it wasn't really necessary anymore.

https://github.com/commercetools/merchant-center-application-kit/pull/89/commits/c5e70b4a863360b4d13c2598d5e7f31d75143bed